### PR TITLE
Add release notes for 1.13.0. Update API docs.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: docs/source/release-history.rst [hunk 1]
```diff
@@ -2,6 +2,56 @@
 Release History
 ***************
 
+v1.13.0 (2019-01-21)
+====================
+
+Added
+-----
+
+* The :class:`~event_model.Filler` accepts an optional paramter ``coerce`` that
```
Comment (jklynch) [inline docs/source/release-history.rst]: "parameter"

File: docs/source/release-history.rst [hunk 2]
```diff
@@ -2,6 +2,56 @@
...
+  can be used to change the behavior of the handlers. This is useful for
+  forcing the filled data to be an in-memory numpy array of a dask array, for
```
Comment (jklynch) [inline docs/source/release-history.rst]: "or"?

File: docs/source/release-history.rst [hunk 3]
```diff
@@ -2,6 +2,56 @@
...
+
+* Field-level filtering in :class:`~event_model.Filler` via the parameters
+  ``include`` and ``exclude`` is deprecated.
+
+Internal Changes
+----------------
+
+* The code in :class:`~event_model.DocumentRouter` that dispatches based on
+  document type has been factored out of ``__call__`` into a new internal
+  method, ``_dispatch``, which makes it easier to for subclasses to modify
```
Comment (mrakitin) [inline docs/source/release-history.rst]: `which makes it easier to for subclasses to modify` -> `which makes it easier for subclasses to modify`

File: docs/source/release-history.rst [hunk 4]
```diff
@@ -2,6 +2,56 @@
...
+  ``__call__`` but reuse the dispatch logic.
+
+See the
+`GitHub v.13.0 milestone <https://github.com/bluesky/event-model/milestone/3>`_
```
Comment (mrakitin) [inline docs/source/release-history.rst]: Should it be v1.13.0?

**Comment by danielballan** [comment]
Thanks @mrakitin. I pushed these copyedits to master as 8b169493c36309c47570de541364d4cc177dd463.

**Pull request merged**